sklearn/ensemble/_hist_gradient_boosting/gradient_boosting.py

```diff
@@ -655,7 +655,14 @@ def _raw_predict(self, X):
             dtype=self._baseline_prediction.dtype
         )
         raw_predictions += self._baseline_prediction
-        for predictors_of_ith_iteration in self._predictors:
+        self._predict_iterations(
```

Comment:
it's not you, but I just realized the shape of raw_predictions in the docstring above is incorrect: it should be 

`raw_predictions : ndarray of shape (n_trees_per_iteration, n_samples)`

would you mind fixing it?